src/containers/Review/notes/NotesTab.tsx

```diff
@@ -131,7 +134,9 @@ const NotesTab: React.FC<{
                     <strong>{note.user?.fullName}</strong>
                   </p>
                   <p className="slightly-smaller-text dark-grey">
-                    {DateTime.fromISO(note.timestamp).toLocaleString(DateTime.DATETIME_MED)}
+                    {DateTime.fromISO(note.timestamp)
+                      .setLocale(locale)
+                      .toLocaleString(DateTime.DATE_FULL)}
```

Comment:
Shouldn't it still be `DateTime.DATETIME_MED` ?

Reply:
No because it shows dates as Sept. or Oct. instead of the full "Setembro" as we normally use in other languages. And it also looks better this way - in my opinion.

Reply:
Okay, that's fine -- just make sure it looks fine in NZ/English as well. 👍